frontend/public/vendor.scss

```diff
@@ -5,31 +5,31 @@
-@import "~bootstrap-sass/assets/stylesheets/bootstrap/type";
-@import "~bootstrap-sass/assets/stylesheets/bootstrap/code";
-@import "~bootstrap-sass/assets/stylesheets/bootstrap/grid";
-@import "~bootstrap-sass/assets/stylesheets/bootstrap/tables";
-@import "~bootstrap-sass/assets/stylesheets/bootstrap/forms";
-@import "~bootstrap-sass/assets/stylesheets/bootstrap/buttons";
-@import "~bootstrap-sass/assets/stylesheets/bootstrap/component-animations";
-@import "~bootstrap-sass/assets/stylesheets/bootstrap/button-groups";
-@import "~bootstrap-sass/assets/stylesheets/bootstrap/list-group";
-@import "~bootstrap-sass/assets/stylesheets/bootstrap/close";
-@import "~bootstrap-sass/assets/stylesheets/bootstrap/modals";
-@import "~bootstrap-sass/assets/stylesheets/bootstrap/utilities";
-@import "~bootstrap-sass/assets/stylesheets/bootstrap/responsive-utilities";
-@import "~bootstrap-sass/assets/stylesheets/bootstrap/dropdowns"; // required by @patternfly/react-console
-@import "~patternfly/dist/sass/patternfly/mixins";
-@import '~patternfly/dist/sass/patternfly/buttons';
-@import '~patternfly/dist/sass/patternfly/dropdowns'; // required by @patternfly/react-console
-@import '~patternfly/dist/sass/patternfly/modals';
-@import '~patternfly/dist/sass/patternfly/forms';
+// @import '~patternfly/dist/sass/patternfly/variables';
+// @import "~bootstrap-sass/assets/stylesheets/bootstrap/variables";
+// @import "~bootstrap-sass/assets/stylesheets/bootstrap/mixins";
+// @import '~patternfly/dist/sass/patternfly/bootstrap-mixin-overrides';
+// @import "~bootstrap-sass/assets/stylesheets/bootstrap/normalize";
+// @import "~bootstrap-sass/assets/stylesheets/bootstrap/scaffolding";
+// @import "~bootstrap-sass/assets/stylesheets/bootstrap/type";
+// @import "~bootstrap-sass/assets/stylesheets/bootstrap/code";
+// @import "~bootstrap-sass/assets/stylesheets/bootstrap/grid";
+// @import "~bootstrap-sass/assets/stylesheets/bootstrap/tables";
+// @import "~bootstrap-sass/assets/stylesheets/bootstrap/forms";
+// @import "~bootstrap-sass/assets/stylesheets/bootstrap/buttons";
+// @import "~bootstrap-sass/assets/stylesheets/bootstrap/component-animations";
+// @import "~bootstrap-sass/assets/stylesheets/bootstrap/button-groups";
+// @import "~bootstrap-sass/assets/stylesheets/bootstrap/list-group";
+// @import "~bootstrap-sass/assets/stylesheets/bootstrap/close";
+// @import "~bootstrap-sass/assets/stylesheets/bootstrap/modals";
+// @import "~bootstrap-sass/assets/stylesheets/bootstrap/utilities";
+// @import "~bootstrap-sass/assets/stylesheets/bootstrap/responsive-utilities";
+// @import "~bootstrap-sass/assets/stylesheets/bootstrap/dropdowns"; // required by @patternfly/react-console
+// @import "~patternfly/dist/sass/patternfly/mixins";
+// @import '~patternfly/dist/sass/patternfly/buttons';
+// @import '~patternfly/dist/sass/patternfly/dropdowns'; // required by @patternfly/react-console
+// @import '~patternfly/dist/sass/patternfly/modals'; // NO LONGER USED I THINK...
```

Comment:
This is definitely still in use (e.g. `modal-header`, `modal-body`, etc.).  https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/_modals.scss

Reply:
I believe https://github.com/openshift/console/pull/9532/files#diff-bad9c904629ba04e3488cfdaa52494fbed20ead152be571e2a351925fd22f55f now contains all the necessary rules to carry over and remove that partial.